internal/pkg/cli/env_show.go

```diff
@@ -151,7 +151,6 @@ func BuildEnvShowCmd() *cobra.Command {
 		GlobalOpts: NewGlobalOpts(),
 	}
 	cmd := &cobra.Command{
-		Hidden: true, //TODO remove when ready for production!
```

Comment:
Are there any other templates where this command needs to be added, or were those issues specific to storage requiring us to enable a new command group?

Reply:
yup that was unique "storage init" if the group already exists nothing needs to be done